### DISCUSSION TITLE
p/664f86d038744a01894699e368cc2708/

### DISCUSSION THREAD

**Discussion by giscus** [rxliuli/blog#20] [Announcements]
# p/664f86d038744a01894699e368cc2708/

前端资源管理 框架及社区 react: 前端流行的 mvc 框架 react-router: react 上的路由管理库 react-router-config: 集中式的 react-router 路由管理库   react-use: react hooks 通用库 antd: 极其强大的中后台 ui 组件库 mobx: 跨框架可变式的全局状态管理库 immer: 简化 react 中不可变数

https://blog.rxliuli.com/p/664f86d038744a01894699e368cc2708/

**Comment by miaopass-future** [comment]
赞，作为一个后端coder最近正在学习前端知识

话说大佬对最近的鹅厂前端大佬被裁---'前端已死'，有没有啥看法呢

**Reply by rxliuli** [reply]
没有关注过，上下文链接是？

**Reply by miaopass-future** [reply]
https://www.zhihu.com/question/589260276/answer/2935047430，之前上过百度热搜的

**Reply by rxliuli** [reply]
吾辈不使用百度没留意到。。。

**Reply by rxliuli** [reply]
没看懂一个人被裁为什么会导致前端已死

**Reply by miaopass-future** [reply]
可能和最近前端系工作就业压力比较大，加之据说被裁是技术系大佬，我有看到脉脉上一群前端改名‘前端已死’。具体原因不明
